SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -306,6 +309,27 @@ void USpatialSender::CreateEntityWithRetries(Worker_EntityId EntityId, FString E
 	Receiver->AddCreateEntityDelegate(RequestId, MoveTemp(Delegate));
 }
 
+void USpatialSender::UpdateWorkerEntityInterestAndPosition()
+{
+	check(Connection != nullptr);
+	check(NetDriver != nullptr);
+	if (NetDriver->WorkerEntityId == SpatialConstants::INVALID_ENTITY_ID)
+	{
+		// No worker entity to update.
+		return;
+	}
+
+	// Update the interest. If it's ready, also adds interest according to the load balancing strategy.
+	FWorkerComponentUpdate InterestUpdate = InterestFactory::CreateServerWorkerInterest(*NetDriver).CreateInterestUpdate();
+	Connection->SendComponentUpdate(NetDriver->WorkerEntityId, &InterestUpdate);
+
+	if (NetDriver->LoadBalanceStrategy->IsReady())
+	{
+		// Also update the position of thew worker entity to the centre of the load balancing region.
+		SendPositionUpdate(NetDriver->WorkerEntityId, NetDriver->LoadBalanceStrategy->GetWorkerEntityPosition());
+	}
+}
+
```

Comment:
till we have dynamic lb strategies I don't think this method is needed

Reply:
I'm not sure what you mean, this is the function which sets interest and updates the worker entity position, we need to call it once